backends/platform/ios7/ios7_osys_main.cpp

```diff
@@ -300,6 +296,19 @@ void OSystem_iOS7::delayMillis(uint msecs) {
 	usleep(msecs * 1000);
 }
 
+GUI::OptionsContainerWidget *OSystem_iOS7::buildBackendOptionsWidget(GUI::GuiObject *boss, const Common::String &name, const Common::String &target) const {
+	return new IOS7OptionsWidget(boss, name, target);
+}
+
+void OSystem_iOS7::registerDefaultSettings(const Common::String &target) const {
+	ConfMan.registerDefault("onscreen_control", false);
+	ConfMan.registerDefault("touchpad_mode", true);
```

Comment:
I thought the original code had some logic around screen dimensions for setting up the default value. Was this lost?

Reply:
I removed that, but I can re-add it.